### PR TITLE
refactor/packages

### PR DESCRIPTION
**Pull request by atoshmukhamedov**

### CODE REVIEW

File: scenes/advertise.ts [hunk 1]
```diff
@@ -6,15 +6,15 @@ import { env } from '#utils/env'
...
+scene.step(async (ctx) => {
   if (1151533771 == ctx.from?.id) {
     await ctx.reply('Give me a message to send every user')
   } else {
     ctx.scene.exit()
   }
 })
 
-scene.wait().on('message:text', async (ctx) => {
+scene.wait('hadith').on('message:text', async (ctx) => {
```
Comment (Xayrulloh) [inline scenes/advertise.ts:17]: What does hadith

File: scenes/donate.ts [hunk 2]
```diff
@@ -6,15 +6,15 @@ import { env } from '#utils/env'
...
+scene.step(async (ctx) => {
   const message = HLanguage(ctx.user.language, 'donateMessage')
 
   ctx.session.message = message
 
   await ctx.reply(message)
 })
 
-scene.wait().on('message:text', async (ctx) => {
+scene.wait('amount').on('message:text', async (ctx) => {
```
Comment (Xayrulloh) [inline scenes/donate.ts:17]: Same

File: scenes/fasting.ts [hunk 3]
```diff
@@ -23,7 +23,7 @@ scene.do(async (ctx) => {
   await ctx.reply(message, { reply_markup: buttons })
 })
 
-scene.wait().on('callback_query:data', async (ctx) => {
+scene.wait('fasting').on('callback_query:data', async (ctx) => {
```
Comment (Xayrulloh) [inline scenes/fasting.ts:26]: Same

File: scenes/hadith.ts [hunk 4]
```diff
@@ -44,7 +44,7 @@ scene.wait().on(['message:text', 'callback_query:data'], async (ctx) => {
     category,
   })
 
-  ctx.reply('Hadith wrote thank you. You are doing your best')
+  await ctx.reply('The hadith is written, thank you. You are doing your best :)')
```
Comment (Xayrulloh) [inline scenes/hadith.ts:47]: Emotional damage

File: scenes/language.ts [hunk 5]
```diff
@@ -41,8 +41,8 @@ scene.wait().on('callback_query:data', async (ctx) => {
   const keyboardText = HLanguage(language, 'mainKeyboard')
   const buttons = customKFunction(2, ...keyboardText)
 
-  ctx.deleteMessage()
-  ctx.reply(message, { reply_markup: { keyboard: buttons.build(), resize_keyboard: true } })
+  await ctx.deleteMessage()
```
Comment (Xayrulloh) [inline scenes/language.ts:44]: Where it came from

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: scenes/language.ts (hunk 5 above)
Comment (Xayrulloh) [reply scenes/language.ts:44]: Esladm

Comment (atoshmukhamedov) [reply scenes/language.ts:44]: man prosto await qo'shganman

File: scenes/hadith.ts (hunk 4 above)
Comment (atoshmukhamedov) [reply scenes/hadith.ts:47]: ?

File: scenes/advertise.ts (hunk 1 above)
Comment (atoshmukhamedov) [reply scenes/advertise.ts:17]: endi wait argument qabul qilar ekan

File: scenes/hadith.ts (hunk 4 above)
Comment (Xayrulloh) [reply scenes/hadith.ts:47]: Hadith wrote => hadith written

File: scenes/advertise.ts (hunk 1 above)
Comment (Xayrulloh) [reply scenes/advertise.ts:17]: Nma qiladi uni

Comment (atoshmukhamedov) [reply scenes/advertise.ts:17]: Screenshot 2023-09-25 at 08 18 26

Comment (atoshmukhamedov) [reply scenes/advertise.ts:17]: umuman oganda farqi yo'q unga unique bo'sa bo'ldi. Meaningful bo'lishi uchun nimani kutvotganizi yozasiz.

File: scenes/hadith.ts (hunk 4 above)
Comment (atoshmukhamedov) [reply scenes/hadith.ts:47]: what is wrong?

Comment (Xayrulloh) [reply scenes/hadith.ts:47]: Nothing, never mind

**Pull request merged**